index.html

```diff
@@ -43,10 +51,10 @@ <h1 class="hero__hed">{{ page.header | escape }}</h1>
           <li>Are the sources of political spending local?</li>
         </ul>
         <div class="hero__funds__raised__header">
-          <h2>Funds Raised in Oakland:</h2>
+          <h2>Total contributions flowing into Oakland's 2018 Election:</h2>
```

Comment:
Do you mind using "smartquotes" here, an apostrophe instead of a straight quote? `’`